src/core/scene/a-scene.js

```diff
@@ -309,6 +309,18 @@ module.exports.AScene = registerElement('a-scene', {
 
         // Callback that happens on enter VR success or enter fullscreen (any API).
         function enterVRSuccess () {
+          // vrdisplaypresentchange fires only once when the first requestPresent is completed;
+          // the first requestPresent could be called from ondisplayactivate and there is no way
+          // to setup everything from there. Thus, we need to emulate another vrdisplaypresentchange
+          // for the actual requestPresent. Need to make sure there are no issues with firing the
+          // vrdisplaypresentchange multiple times.
+          var event = new CustomEvent('vrdisplaypresentchange', {
```

Comment:
One more thing. Styling. We can maybe combine in one line:

 `var event = new CustomEvent('vrdisplaypresentchange', {detail: {display: utils.device.getVRDisplay()}});`

Reply:
Yeah, linter just argued about inconsistent spacing, so I decided to make sure that spacing is super consistent! ;) but no biggie, can make it one-liner.

Reply:
our linter doesn't like space around curly braces. e.g `{default: true}`